Sources/Commands/SwiftTool.swift

```diff
@@ -413,10 +413,13 @@ public class SwiftTool {
             diagnostics.emit(error: "'--netrc-file' option is only supported on macOS >=10.13")
             #endif
         }
-        
+
+        // --enable-test-discovery should never be called on darwin based platforms
+        #if canImport(Darwin)
```

Comment:
Shouldn't this be deleted? Otherwise there are two warnings now, one from here and one from `BuildPlan` or am I missing something?

Reply:
the other code (BuildPlan) would only run on non-drawin platforms

Reply:
Right, sorry, I somehow misread this condition as saying not-Darwin